src/appShell/App/Container.tsx

```diff
@@ -65,6 +62,27 @@ export default class Container extends React.Component<IContainerProps, {}> {
             );
         }
 
+        if (ServerConfigHelpers.oncoKbIsEnabledWithoutToken()) {
+            return (
+                <div className="contentWrapper">
+                    <ErrorScreen
+                        title={'OncoKB token is not available'}
```

Comment:
Should we have longer title like `OncoKB is enabled but token is not available`?

Reply:
yes that would be nice